packages/server/lib/gui/windows.ts

```diff
@@ -185,7 +185,7 @@ export function create (projectRoot, _options: WindowOptions, newBrowserWindow =
     })
   }
 
-  win.webContents.on('crashed', function (...args) {
+  win.webContents.on('render-process-gone', function (...args) {
```

Comment:
Addresses this type failure: https://cloud.nx.app/runs/cibLuMvwic
![Screenshot 2024-09-12 at 2 58 31 PM](https://github.com/user-attachments/assets/cf24350a-d80a-4714-95d7-1fcf1beb9eba)